test/Extensions/ExtensionTests.jl

```diff
@@ -3,16 +3,19 @@ module ExtensionTests
 "!!set(name:test)!!get(name)"
 set_and_get = 1
 
+"!!setget(name:test)"
+setget = 2
+
 "!!summary(summary)"
-summary = 2
+summary = 3
 
 "!!longform(...)"
-longform = 3
+longform = 4
 
 "!!include(includes/file.md)"
-includes = 4
+includes = 5
 
 "!!set(name:test) !!include(includes/file.md) !!get(name)"
-set_includes_get = 5
+set_includes_get = 6
```

Comment:
The numbering wasn't meant to be significant. Perhaps we should replace the numbers with `()` to avoid further renumbering later?